src/QsCompiler/LanguageServer/Program.cs

```diff
@@ -83,6 +83,11 @@ private static int LogAndExit(ReturnCode code, string? logFile = null, string? m
 
         public static int Main(string[] args)
         {
+            // We need to set the current directory to the same directory of
+            // the LanguageServer executable so that it will pick the global.json file
+            // and force the MSBuildLocator to use a particular version of .NET Core SDK
+            Directory.SetCurrentDirectory(AppDomain.CurrentDomain.BaseDirectory);
+
```

Comment:
Since the language server doesn't have a `global.json` anymore, I think this can be removed?

```suggestion
```